Libraries/src/Amazon.Lambda.AspNetCoreServer.Hosting/ServiceCollectionExtensions.cs

```diff
@@ -88,6 +88,61 @@ public static IServiceCollection AddAWSLambdaHosting(this IServiceCollection ser
             return services;
         }
 
+        #if NET8_0_OR_GREATER
+        /// <summary>
+        /// Adds a <see cref="HttpRequestMessage"/>> that will be used to invoke
+        /// Routes in your lambda function in order to initialize the ASP.NET Core and Lambda pipelines
+        /// during <see cref="SnapshotRestore.RegisterBeforeSnapshot"/>.  This improves the performance gains
+        /// offered by SnapStart.
+        /// <para />
+        /// The returned <see cref="HttpRequestMessage"/> must have a relative
```

Comment:
nit: Remove the word "returned". I assume this is leftover when this was a `Func`

Reply:
fixed